acceptance/testdata/launcher/Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN go build -o helper ./src/exec.d
 
```

Comment:
Does it make sense to rename this file to `Dockerfile.linux` to keep the convention with `Dockerfile.windows`?
Same comment applies to other files that were renamed.

Reply:
I took this pattern from the `tools/` directory.

Reply:
(not sure if Dockerfile.linux would be better but just wanting to keep a single convention where possible!)